k8s/runners/protected/pcluster/release.yml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: runner-x86-pcluster-prot
+  namespace: gitlab
+spec:
+  interval: 10m
+  url: https://charts.gitlab.io
+
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: runner-x86-pcluster-prot
+  namespace: gitlab
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: gitlab-runner
+      version: 0.37.2 # gitlab-runner@14.7.0
+      sourceRef:
+        kind: HelmRepository
+        name: runner-x86-pcluster-prot
+  values:
+    image: gitlab/gitlab-runner:alpine-v14.7.0
+    imagePullPolicy: IfNotPresent
+    replicas: 6
+
+    gitlabUrl: "https://gitlab.spack.io/"
+    unregisterRunners: true
+    terminationGracePeriodSeconds: 21600 # six hours
+    concurrent: 20
+    checkInterval: 30
+
+    metrics:
+      enabled: true
+
+    rbac:
+      serviceAccountName: runner
+
+    runners:
+      config: |
+        [[runners]]
+          name = "gitlab-aws-pcluster-autoscaler-prot"
+          url = "gitlab.spack.io"
+          executor = "docker+machine"
+          limit = 20
+          output_limit = 4096
+          [runners.machine]
+            IdleCount = 1
+            IdleTime = 900
+            MaxBuilds = 100
+            MachineDriver = "amazonec2"
+            MachineName = "gitlab-ci-pcluster-protected-%s"
+            MachineOptions = [
+              "amazonec2-ami=ami-011244318c573c915", # aws-parallelcluster-3.5.1-amzn2-hvm-x86_64-202303171148 2023-03-17T11-52-52.966Z
+              "amazonec2-region=us-east-1",
+              "amazonec2-use-private-address=true",
+              "amazonec2-tags=runner-manager-name,gitlab-aws-autoscaler,gitlab,true,gitlab-runner-autoscale,true",
+              "amazonec2-instance-type=t3.medium",
+              "amazonec2-iam-instance-profile=BEEF",
```

Comment:
Need to figure out what this needs to be

Reply:
There is also a field for key-pairs. I think both of these can be expressed as environment variables, is it possible to  populate those via secrets here?

@AlmightyYakob